app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+import os
+from typing import Callable
+
+
+class CommandError(Exception):
+    """Checks command input for right format."""
+
+
+def check_command(func: Callable) -> Callable:
+    def wrapper(command: str) -> Callable:
+        if command.split()[0] != "mv" or len(command.split()) != 3:
+            raise CommandError("Please enter valid command!")
+        return func(command)
+
+    return wrapper
+
+
+@check_command
+def move_file(command: str) -> None:
+    source_name, target_name = command.split()[1:3]
+    if "/" not in target_name:
+        os.rename(source_name, target_name)
```

Comment:
Instead of relying on a hardcoded '/' as a directory separator in the script, consider using a platform-agnostic approach to ensure that the destination path includes the necessary folders